tests/sentry/seer/similarity/test_utils.py

```diff
@@ -634,6 +634,22 @@ def test_chained_too_many_frames_minified_js_frame_limit(self):
                 == expected_frame_count
             )
 
+    def test_chained_too_many_exceptions(self):
+        """Test that we restrict number of chained exceptions to 30."""
+        data_chained_exception = copy.deepcopy(self.CHAINED_APP_DATA)
+        data_chained_exception["app"]["component"]["values"][0]["values"] = [
+            self.create_exception(
+                exception_type_str="Exception",
+                exception_value=f"value {i}!",
```

Comment:
Nit: This makes it clearer down in  the assertions what we're asserting about.

```suggestion
                exception_value=f"exception {i} message!",
```